packages/story-editor/src/components/library/paneIds.js

```diff
@@ -14,5 +14,11 @@
  * limitations under the License.
  */
 
-export { default as ThumbnailWrapper } from './thumbnailWrapper';
-export { default as VideoChecklistCard } from './videoChecklistCard';
+export const PANE_IDS = {
+  MEDIA: 'media',
+  MEDIA_3P: 'media3p',
+  TEXT: 'text',
+  SHAPES: 'shapes',
+  ELEMENTS: 'elements',
+  PAGE_TEMPLATES: 'pageTemplates',
+};
```

Comment:
This is a weird diff. Two things happened here:

1. Broke a loop that involved the checklist cards and the thumbnail wrapper + video checklist cards: 566a1b3 and 083b38b. The file in red was deleted.
2. Broke a loop that involved the highlights provider: c5dd4c68. The file in green was added.

These two are not related.